test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1089,6 +1092,39 @@ void UcxConnection::set_log_prefix(const struct sockaddr* saddr,
     memcpy(_log_prefix, ss.str().c_str(), length);
 }
 
+void UcxConnection::set_verbose_log_prefix()
```

Comment:
why set_log_prefix is not enough?
if we want more info for a connection, maybe we can unite them to a single function so it will always be printed?

Reply:
`set_verbose_log_prefix(`) needs to be called after ep established because it needs query source address from an established ep. so `set_log_prefix()` is called as it was (i.e., before ep is established) and `set_verbose_log_prefix()` called after ep is established if -V specified.

Reply:
i think this creates too much code duplication ; why need to print both source and destination address? we already have connection number to track it.
If really need to print both, we need to unite these functions: decide on a _single_ prefix format, have a _single_ function to set it; potentially some information in this prefix will be stub such as 0.0.0.0 before it's fully established.